mmdet/evaluation/metrics/openimages_metric.py

```diff
@@ -1,237 +1,185 @@
 # Copyright (c) OpenMMLab. All rights reserved.
-import copy
-from collections import OrderedDict
-from typing import List, Optional, Sequence, Union
+from typing import List, Optional, Sequence, Tuple, Union
 
 import numpy as np
-from mmengine.evaluator import BaseMetric
-from mmengine.logging import MMLogger, print_log
+from mmengine.logging import print_log
+from mmeval import OIDMeanAP
+from terminaltables import AsciiTable
 
 from mmdet.registry import METRICS
-from ..functional import eval_map
 
 
 @METRICS.register_module()
-class OpenImagesMetric(BaseMetric):
-    """OpenImages evaluation metric.
+class OpenImagesMetric(OIDMeanAP):
```

Comment:
The base class is changed. Will this cause some bugs in MMEngine?

Reply:
Right now, it won't cause bugs.